src/object/srv_obj_migrate.c

```diff
@@ -1301,6 +1399,53 @@ migrate_cont_iter_cb(daos_handle_t ih, d_iov_t *key_iov,
 	}
 
 
+	/* Check if the container uuid is in the init hash table
+	 * If it is, this container has already had at least one object in it
+	 * migrated during this cycle, so it does not need to be created
+	 *
+	 * If this migration process is part of reintegration, the container
+	 * should first be deleted before being re-created
+	 *
+	 * TODO: Hook up reintegration boolean
+	 */
+	link = d_hash_rec_find(&tls->mpt_cont_init_tab, cont_uuid,
+			       sizeof(uuid_t));
+	if (!link) {
+		/* Not actually storing anything in the table - just using it
+		 * to test set membership. The link stored is just the simplest
+		 * base list type
+		 */
+		d_list_t *rlink;
+
+		D_DEBUG(DB_TRACE,
+			"Migration ensure container "DF_UUID" created\n",
+			DP_UUID(cont_uuid));
+
+		//rc = migrate_cont_ensure_created(tls, cont_uuid, false);
+		//if (rc)
+		//	D_GOTO(free, rc);
+
+		/* Insert a link into the hash table to mark this cont_uuid as
+		 * having already been initialized
+		 */
+		D_ALLOC_PTR(rlink);
+		if (rlink == NULL)
+			D_GOTO(free, rc=-DER_NOMEM);
```

Comment:
(style)  spaces required around that '=' (ctx:VxO)
(style)  space required before that '-' (ctx:OxV)